### PR TITLE
Fixed News SEO sitemap not being generated when News post type exists

### LINKED ISSUE #114: Post-type "News" conflicts with news sitemap.
Please use this filter to fix https://github.com/Yoast/wpseo-news/blob/trunk/classes/class-sitemap.php#L37

### PR DESCRIPTION
**Pull request by jcomack**
Fixed bug that caused the News SEO sitemap to not be generated when a post type exists with the same name.

Fixes #114

### CODE REVIEW

File: classes/class-sitemap.php [hunk 1]
```diff
@@ -200,6 +205,49 @@ private function get_post_types() {
...
+
+	/**
+	 * Getting the name for the sitemap, if $full_path is true, it will return the full path
+	 *
+	 * @param bool $full_path
+	 *
+	 * @return string mixed
+	 */
+	public static function get_sitemap_name( $full_path = true ) {
+		// This filter is documented in classes/class-sitemap.php
```
Comment (andizer) [inline classes/class-sitemap.php]: All inline comments must end with a full stop (`.`)

File: classes/class-sitemap.php [hunk 2]
```diff
@@ -200,6 +205,49 @@ private function get_post_types() {
...
+		$sitemap_name = apply_filters( 'wpseo_news_sitemap_name', self::news_sitemap_basename() );
+
+		// When $full_path is true, it will generate a full path
```
Comment (andizer) [inline classes/class-sitemap.php]: All inline comments must end with a full stop (`.`)

**Comment by andizer** [comment]
CR 1 done.  Please fix the merge conflicts.

I think the codestyle validation isn't up-to-date, because there are some codestyle things not being reported by travis. Probably we have to care about these later.

**Comment by andizer** [comment]
I've updated this pr to make sure the right stylesheet is loaded.

**Comment by terw-dan** [comment]
CR done :+1:

**Comment by terw-dan** [comment]
Acceptance :+1:

**Pull request merged**